docs/security.md

```diff
@@ -48,7 +48,7 @@ By default AnonymousAuth on the kubelet is 'on' and so communication between kub
 # In the cluster spec
 spec:
   kubelet:
-    anonymousAuth: false
+    AnonymousAuth: false
```

Comment:
pretty sure this should remain anonymousAuth not AnonymousAuth

Reply:
I hadn't looked at `componentconfig.go`, will revert this change.